src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java

```diff
@@ -2226,17 +2239,35 @@ protected String extractAuthorName(Document doc) {
                     }
 
                     // http://nation.foxnews.com/2017/06/13/sessions-inquisition-comes-empty-trump-unscathed-after-attorney-generals-testimony
-                    if (matches == null || matches.size() == 0){
+                    if (matches == null || matches.size() == 0) {
```

Comment:
Please remove this rule as requested in this PR: https://github.com/skyshard/snacktory/pull/42